src/Controllers/UserStoryController.php

```
@@ -495,6 +513,45 @@ public function update($id): void
         $this->response->redirect('/app/user-stories?project_id=' . $story['project_id']);
     }
 
+    /**
+     * Return edit-only story fields without duplicating them into every list row.
+     *
+     * @param int $id User story primary key (from route parameter)
+     */
+    public function editData($id): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $id. Configured minimum length is 3. (undefined)

(ShortVariable)